variants/NUCLEO_L552ZE_Q/variant.h

```diff
@@ -0,0 +1,289 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2020, STMicroelectronics
+ * All rights reserved.
+ *
+ * This software component is licensed by ST under BSD 3-Clause license,
+ * the "License"; You may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *                        opensource.org/licenses/BSD-3-Clause
+ *
+ *******************************************************************************
+ */
+
+#ifndef _VARIANT_ARDUINO_STM32_
+#define _VARIANT_ARDUINO_STM32_
+#ifdef __cplusplus
+extern "C" {
+#endif // __cplusplus
+
+/*----------------------------------------------------------------------------
+ *        STM32 pins number
+ *----------------------------------------------------------------------------*/
+#define PD9  0 // USART3_RX
+#define PD8  1 // USART3_TX
+#define PF15 2
+#define PE13 3
+#define PF14 4
+#define PE11 5
+#define PE9  6
+#define PF13 7
+#define PF12 8
+#define PD15 9
+#define PD14 10
+#define PA7  11 // SPI_A_MOSI
+#define PA6  12 // SPI_A_MISO
+#define PA5  13 // SPI_A_SCK
+#define PB9  14 // I2C_A_SDA
+#define PB8  15 // I2C_A_SCL
+#define PC6  16
+#define PD11 17
+#define PB13 18
+#define PD12 19
+#define PA4 20
+#define PB4 21
+#define PB5 22 // SPI_B_MOSI
+#define PB3 23 // SPI_B_SCK
+#define PA4 24 // SPI_B_NSS
+#define PB4 25 // SPI_B_MISO
```

Comment:
Multiple definition, several warning about that during the compile
// 24 is PA4                   (20)
// 25 is PB4                   (21)

Other pins are also duplcated